src/Containers/MinimalContainers/ConstantSizeMatrix.hpp

```diff
@@ -181,9 +194,8 @@ class ConstantSizeMatrix
   size_t n_;
   size_t m_max_;
   size_t n_max_;
+  size_t capacity_;
```

Comment:
is `capacity` always equal to `data_.size()`?

Reply:
Yes, I think so.